lib/light-service/orchestrator.rb

```diff
@@ -86,6 +86,15 @@ def with_callback(action, steps)
         end
       end
 
+      def after_failing(symbols)
+        symbols = [symbols] unless symbols.is_a?(Array)
+        @context.orchestrator_callbacks[:after_failing] =
+          symbols.map do |symbol|
+            method(symbol)
```

Comment:
The `method` method turns a class method into a lambda. Seriously, awesome Ruby......